test/built-ins/Date/prototype/toString/non-date-receiver.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2017 V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: #sec-date.prototype.tostring
+description: Date.prototype.toString throws a TypeError on non-Date receivers
+---*/
```

Comment:
The info tag is being  valuable for reviews (even after the PR phase). If you don't mind, I would like to add the following:

```diff
+info: |
+  Date.prototype.toString ( )
+
+  1. Let tv be ? thisTimeValue(this value).
```